code/datums/loadout/item_representation/item_representation.dm

```diff
@@ -0,0 +1,68 @@
+/**
+ * Light weight representation of an obj/item
+ * This allow us to manipulate and store a lot of item-like objects, without it costing a ton of memory or having to instantiate all items
+ * This also allow to save loadouts with jatum, because it doesn't accept obj/item
+ */
+/datum/item_representation
+	/// The type of the object represented, to allow us to create the object when needed
+	var/item_type
+	/// If it's allowed to bypass the vendor check
+	var/bypass_vendor_check = FALSE
```

Comment:
That's only for internal storage, that are items not in vendor

Reply:
I dont get it and I dont get what vendor check is either